contrib/strongswan/files/sysusers.conf

```diff
@@ -0,0 +1,5 @@
+# Create _strongswan system user
+
+u _strongswan - "ipsec user" /var/empty /usr/bin/nologin
+g _strongswan - "ipsec group"
+m _strongswan _strongswan
```

Comment:
i don't think these two lines are needed, the first one should automatically create a _strongswan group and add the _strongswan user to it

Reply:
You are right.  Fixed.